modules/troubleshooting-common-problems-with-distributed-workloads-for-users.adoc

```diff
@@ -0,0 +1,234 @@
+:_module-type: REFERENCE
+
+[id="troubleshooting-common-problems-with-distributed-workloads-for-users_{context}"]
+= Troubleshooting common problems with distributed workloads for users
+
+[role='_abstract']
+If you are experiencing errors in {productname-long} relating to distributed workloads, read this section to understand what could be causing the problem, and how to resolve the problem.
+
+ifndef::upstream[]
+If you cannot see the problem here or in the release notes, contact {org-name} Support.
+endif::[]
+
+== My Ray cluster is in a suspended state
+
+.Problem
+The resource quota specified in the cluster queue configuration might be insufficient, or the resource flavor might not yet be created.
+
+.Diagnosis
+The Ray cluster head pod or worker pods remain in a suspended state. 
+Check the status of the `Workloads` resource that is created with the `RayCluster` resource.
+The `status.conditions.message` field provides the reason for the suspended state, as shown in the following example:
+
+[source,bash]
+----
+status:
+ conditions:
+   - lastTransitionTime: '2024-05-29T13:05:09Z'
+     message: 'couldn''t assign flavors to pod set small-group-jobtest12: insufficient quota for nvidia.com/gpu in flavor default-flavor in ClusterQueue'
+
+----
+
+.Resolution
+Check your cluster queue configuration to ensure that the resources that you requested are within the limits defined for the project.
+Either reduce your requested resources, or contact your administrator to request more resources. 
+
+== My Ray cluster is in a failed state
+
+.Problem
+You might have insufficient resources.
+
+.Diagnosis
+The Ray cluster head pod or worker pods are not running.
+When a Ray cluster is created, it initially enters a `failed` state. 
+This failed state usually resolves after the reconciliation process completes and the Ray cluster pods are running.
+
+.Resolution
+If the failed state persists, review the pod events to identify the cause of the problem.
```

Comment:
Wondering if the word "events" here (and throughout the doc) should be capitalised as well, given it's a tab option the user can click and view. - I don't have a strong opinion on this.

Reply:
We're referring to the events, rather than the tab, so we don't need to use initcap.

Reply:
I added some text to indicate that the events are on the **Events** tab.